blocks/hero-block/features/hero/default.jsx

```diff
@@ -1,30 +1,38 @@
 import React, { Fragment } from "react";
 import PropTypes from "@arc-fusion/prop-types";
+import { RESIZER_TOKEN_VERSION, RESIZER_URL } from "fusion:environment";
```

Comment:
`RESIZER_URL` in no longer in the environment variables.

Reply:
Removed `RESIZER_URL`